exercises/bowling/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+pub struct BowlingGame {
+}
+
+#[derive(Debug, PartialEq)]
+pub enum Error {
```

Comment:
Tiny nitpick: I prefer to group structs and their impls together; this enum should therefore be defined either before or after the `BowlingGame` section is complete.

This issue is of minimal importance to me; I do not mind if it is ignored, and it will not affect my review.